Assets/Scripts/Game/UserInterfaceWindows/DaggerfallUnityMouseControlsWindow.cs

```diff
@@ -123,15 +123,15 @@ protected override void Setup()
             SetupKeybindButton(quickSaveKeybindButton, InputManager.Actions.QuickSave, 210, 20);
             SetupKeybindButton(quickLoadKeybindButton, InputManager.Actions.QuickLoad, 210, 40);
 
-            mouseSensitivitySlider = CreateSlider("Mouse Look Sensitivity", 15, 80, 0.1f, 8.0f, DaggerfallUnity.Settings.MouseLookSensitivity);
-            invertMouseVerticalCheckbox = AddOption(20, 100, "Invert Look-Y", DaggerfallUnity.Settings.InvertMouseVertical);
-            mouseSmoothingCheckbox = AddOption(20, 110, "Mouse Smoothing", DaggerfallUnity.Settings.MouseLookSmoothing);
-            moveSpeedCheckbox = AddOption(20, 120, "Movement Acceleration", DaggerfallUnity.Settings.MovementAcceleration);
+            mouseSensitivitySlider = CreateSlider("Mouse Look Sensitivity", 15, 100, 0.1f, 8.0f, DaggerfallUnity.Settings.MouseLookSensitivity);
+            invertMouseVerticalCheckbox = AddOption(20, 120, "Invert Look-Y", DaggerfallUnity.Settings.InvertMouseVertical);
+            mouseSmoothingCheckbox = AddOption(20, 130, "Mouse Smoothing", DaggerfallUnity.Settings.MouseLookSmoothing);
+            moveSpeedCheckbox = AddOption(20, 140, "Movement Acceleration", DaggerfallUnity.Settings.MovementAcceleration);
 
             //weaponSensitivitySlider = CreateSlider("Mouse Weapon Sensitivity", 115, 80, 0.1f, 10.0f, DaggerfallUnity.Settings.WeaponSensitivity);
-            clickToAttackCheckbox = AddOption(115, 100, "Click to Attack", DaggerfallUnity.Settings.ClickToAttack);
-            bowDrawbackCheckbox = AddOption(115, 110, "Bows - draw and release", DaggerfallUnity.Settings.BowDrawback);
-            toggleSneakCheckbox = AddOption(115, 120, "Toggle Sneak", DaggerfallUnity.Settings.ToggleSneak);
+            attackModeSlider = CreateSlider("Attack Mode", 115, 100, DaggerfallUnity.Settings.AttackMode, "Vanilla", "Click", "Hold");
```

Comment:
From a UI perspective I found the placement of this slider to be a bit awkward. Could you put the two sliders next to each other above, and then the mouse weapon threshold below them, so it looks like this? (Image was done with Paint):

https://imgur.com/a/irC2ncp